post.md

```diff
@@ -325,50 +325,50 @@ It would get called if the component reanimated successfully then unmounted, but
 For example if `Albums` started to render, made the promise and suspended. Then something in the application changed before the promise resolved, maybe the user navigates away or decides on a different band to look at. `Albums` is never reanimated by suspense, so we would never get that `useEffect` cleanup.
 
 Libraries like [`@tanstack/react-query`](https://tanstack.com/query/latest/docs/react/overview) do this for us, with declared timeouts and cache lengths that we can configure.
-So maybe it makes sense that the react team are pushing for us not to try and work with `Suspense` directly.
+So maybe it makes sense that the React team are pushing for us not to try and work with `Suspense` directly.
 It could lead us into some dangerous, time consuming, territory. As an application developer I want to be writing features, not caching mechanisms.
 
 On the other hand, it turns out that a cache like this isn't necessarily what's required.
 In a [only very slightly more complicated example application](https://github.com/jaybeeuu/react-suspense)
-I managed the promise managed with hooks in a [wrapper](https://github.com/jaybeeuu/react-suspense/blob/main/src/image-loader/image-loader.tsx) between the `Suspense` and the component that needed the data.
+I managed the promise with hooks in a [wrapper](https://github.com/jaybeeuu/react-suspense/blob/main/src/image-loader/image-loader.tsx) between the `Suspense` and the component that needed the data.
 The downside here being the separation between the creation of the data from the need for the data.
-But depending on your application, and philosophy, that could be desirable any way.
+But depending on your application, and philosophy, that could be desirable anyway.
```

Comment:
I feel like that separation has been the philosophy on my last project, so it feels like it's not a problem reading this. But then, we were talking about fetching the data earlier in the post so I want to be really clear here about what we mean by data creation vs data fetching vs needing the data